pkg/scheduler/eventhandlers.go

```diff
@@ -366,6 +366,11 @@ func (sched *Scheduler) skipPodUpdate(pod *v1.Pod) bool {
 		// Annotations must be excluded for the reasons described in
 		// https://github.com/kubernetes/kubernetes/issues/52914.
 		p.Annotations = nil
+		// Same as above, when annotations are modified with ServerSideApply,
+		// ManagedFields may also change and must be excluded
+		p.ManagedFields = nil
```

Comment:
Prefer to remove this out. If it's agreed to back this SSA change, the right way is to backport the original PR (#90373) in a separate PR.

Reply:
We probably don't need to backport that one as far back. Not sure how old Server Side Apply is.

Reply:
SSA was beta in 1.16, but I heard there was some implication in the first beta version which doesn't record which actor modified the fields (or sth. like that), so 🤷‍♂️ not quite sure it's widely adopted back in 1.16.

> We probably don't need to backport that one as far back.

Agree.